HelpSource/Classes/Collection.schelp

```diff
@@ -225,12 +225,24 @@ code::
 List[1, 2, 3, 4].detect({ arg item, i; item.even });
 ::
 
+method::detectLast
+Similar to link::#-detect::, but doing in a reversed manner.
+
 method::detectIndex
 Similar to link::#-detect:: but returns the index instead of the item itself.
 code::
 List[1, 2, 3, 4].detectIndex({ arg item, i; item.even });
 ::
 
+method::detectLastIndex
+Similar to link::#-detectIndex::, but doing in a reversed manner.
+
+method::lastForWhich
+Synonym to link::#-detectLast::, which is added to keep the consistency of naming.
```

Comment:
Could you write instead: "Returns the last element of the collection for which the function is true. Synonym to link::#-detectLast::."

Also why not add an example here – it is the recommended method.